__tests__/app.test.js

```diff
@@ -218,3 +218,66 @@ describe("getAllReviews", () => {
         });
     });
 });
+
+describe("getCommentsByReviewId", () => {
+    test("status 200, getCommentsByReviewId returns an array of comment objects with all properties", () => {
+        const commentsArray = [{
+            comment_id: 1,
+            body: 'I loved this game too!',
+            votes: 16,
+            author: 'bainesface',
+            review_id: 2,
+            created_at: "2017-11-22T12:43:33.389Z",
+          },
+          {
+            comment_id: 4,
+            body: 'EPIC board game!',
+            votes: 16,
+            author: 'bainesface',
+            review_id: 2,
+            created_at: "2017-11-22T12:36:03.389Z",
+          },
+          {
+            comment_id: 5,
+            body: 'Now this is a story all about how, board games turned my life upside down',
+            votes: 13,
+            author: 'mallionaire',
+            review_id: 2,
+            created_at: "2021-01-18T10:24:05.410Z",
+          }]
+        return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
```

Comment:
You could refactor this test to check each comment obj is the "shape" we'd expect them to be. Both approaches are fine - just a suggestion if you wanted to try it.